polyfills/HTMLCanvasElement/protoype/toBlob/config.json

```diff
@@ -0,0 +1,15 @@
+{
+	"browsers": {
+		"chrome": "<50",
```

Comment:
I am not certain for the other browsers. As a general rule, isn't it better to target a restrictive set of browsers and extend it if needed?